CMakeLists.txt

```diff
@@ -131,20 +135,17 @@ set(NOVELRT_EXECUTABLE_HEADERS
 )
 
 set(NOVELRT_EXECUTABLE_SOURCES
+  ${NOVELRT_SOURCES}
   src/main.cpp
 )
 
 set(NOVELRT_EXECUTABLE_LINK_LIBRARIES
-  NovelRTLib
+  ${NOVELRT_LINK_LIBRARIES}
 )
 
 add_executable(NovelRT ${NOVELRT_EXECUTABLE_HEADERS} ${NOVELRT_EXECUTABLE_SOURCES})
 target_link_libraries(NovelRT ${NOVELRT_EXECUTABLE_LINK_LIBRARIES})
 
-if (WIN32)
```

Comment:
This shouldn't be removed, it breaks the `NovelRTLib` build

Reply:
Same here.